spec/features/create_a_summary_document_activity_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+RSpec.describe 'Summary document activity', vcr: true do
+  it 'is successfully created' do
+    summary_document = TelephoneAppointments::SummaryDocumentActivity.new(
+      appointment_id: 35,
+      guider_uid: '7827dce2-df16-4e92-ba69-39c05f27720b'
+    )
+    expect(summary_document.save).to be true
+  end
+
+  it 'it provides a list of errors when the create is unsuccessful' do
+    summary_document = TelephoneAppointments::SummaryDocumentActivity.new(
+      appointment_id: nil,
+      guider_uid: nil
+    )
+
+    expect(summary_document.save).to be false
+    expect(summary_document.errors).to eq(
+      'appointment' => ['can\'t be blank'],
+      'owner' => ['can\'t be blank']
```

Comment:
The response keys for the errors don't match the given arguments so it may be difficult for consumers to tie the two up. This is more of an observation than a hard requirement though, so feel free to pass this over.